internals/generators/container/index.js.hbs

```diff
@@ -21,11 +21,18 @@ import { useSelector, useDispatch } from 'react-redux';
 import { createStructuredSelector } from 'reselect';
 {{/if}}
 
-{{#if wantSaga}}
-import { useInjectSaga } from 'utils/injectSaga';
+{{#if wantActionsAndReducer}}
+  {{#if wantSaga}}
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+  {{else}}
+import { useInjectReducer } from 'redux-injectors';
+  {{/if}}
+{{else}}
+  {{#if wantSaga}}
+import { useInjectSaga } from 'redux-injectors';
+  {{/if}}
 {{/if}}
 {{#if wantActionsAndReducer}}
-import { useInjectReducer } from 'utils/injectReducer';
```

Comment:
Do you mind opening a separate PR if you'd like to update the logic for the container generator?

(I agree that there's an issue if wantActionsAndReducer is false but wantSaga is true but it should be handled separately from this.)

Reply:
By this I mean that please only update the imports here.

Reply:
I could just change the import but then you could up with this:
```
import { useInjectReducer } from 'redux-injectors';
import { useInjectSaga } from 'redux-injectors';
```

Which I'm pretty sure will throw a linting error under https://eslint.org/docs/rules/no-duplicate-imports , and it seems bad to have the generator generate a file with a linting error.

Reply:
Oh my bad! I went over this too fast. The logic remains the same but it looks more complicated because you're importing from one package instead of 2 files.